test/ecdsatest.c

```diff
@@ -8,32 +8,25 @@
  * https://www.openssl.org/source/license.html
  */
 
-#include <stdio.h>
-#include <stdlib.h>
-#include <string.h>
-
-#include <openssl/opensslconf.h> /* To see if OPENSSL_NO_EC is defined */
-#include "testutil.h"
+# include <openssl/opensslconf.h> /* To see if OPENSSL_NO_EC is defined */
+# include "testutil.h"
```

Comment:
nit: these before the `#ifndef` should not be indented with the extra space, i.e. `#include`

Reply:
Thanks! cherry picked some stuff from `openssl-format-source`, then fixed in c51ffb56485860f631540b5b329397eefff23bf6